bridge/client/app/evaluation-board/evaluation-board.component.ts

```diff
@@ -0,0 +1,74 @@
+import {ChangeDetectorRef, Component, OnDestroy, OnInit} from '@angular/core';
+import {filter, map, startWith, switchMap} from "rxjs/operators";
+import {Observable, Subscription, timer} from "rxjs";
+import {ActivatedRoute, Router} from "@angular/router";
+import {Location} from "@angular/common";
+
+import * as moment from 'moment';
+
+import {Root} from "../_models/root";
+import {Project} from "../_models/project";
+
+import {DataService} from "../_services/data.service";
+import {ApiService} from "../_services/api.service";
+import DateUtil from "../_utils/date.utils";
+import {Service} from "../_models/service";
+import {labels, Trace} from "../_models/trace";
+import {Stage} from "../_models/stage";
+import {DtCheckboxChange} from "@dynatrace/barista-components/checkbox";
+
+@Component({
+  selector: 'app-project-board',
+  templateUrl: './evaluation-board.component.html',
+  styleUrls: ['./evaluation-board.component.scss']
+})
+export class EvaluationBoardComponent implements OnInit, OnDestroy {
+
+  public _routeSubs: Subscription = Subscription.EMPTY;
+  public error: string = null;
+
+  public contextId: string;
+  public root: Root;
+  public evaluations: Trace[];
+
+  constructor(private _changeDetectorRef: ChangeDetectorRef, private router: Router, private location: Location, private route: ActivatedRoute, private dataService: DataService, private apiService: ApiService) { }
+
+  ngOnInit() {
+    this._routeSubs = this.route.params.subscribe(params => {
+      if(params["shkeptncontext"]) {
+        this.contextId = params["shkeptncontext"];
+        this.apiService.getTraces(this.contextId)
+          .pipe(
+            map(response => response.body),
+            map(result => result.events||[]),
+            map(traces => traces.map(trace => Trace.fromJSON(trace)).sort((a, b) => new Date(a.time).getTime() - new Date(b.time).getTime()))
+          )
+          .subscribe((traces: Trace[]) => {
```

Comment:
Subscription is never cleaned up

Reply:
I've also mentioned this [here](https://github.com/keptn/keptn/pull/1877#discussion_r446008873) already, should be cleaned up in the entire bridge

Reply:
I created an `unsubscribe$` Subject and use `.pipe(takeUntil(this.unsubscribe$))` before each subscribe.